src/nvme/mi.h

```diff
@@ -89,6 +89,7 @@
 
 #include "types.h"
 #include "tree.h"
+#include <poll.h>
```

Comment:
I don't think we need this? (and if we do, it might indicate a layering violation)